hw/dv/tools/vcs/common_cov_excl.cfg

```diff
@@ -12,5 +12,6 @@
 -node tb.dut *tl_o.d_opcode[2]
 -node tb.dut *tl_o.d_sink
 
-// [LOW_RISK] Verified in prim_alert_receiver TB."
--node tb.dut *alert_rx_*.ping_*
+// [LOW_RISK] Verified in prim_alert_sender/receiver TB."
+-node tb.dut* *alert_rx_i*.ping_p
+-node tb.dut* *alert_rx_i*.ping_n
```

Comment:
@rasmus-madsen could you help check the equivalent expression for xcelium?
https://github.com/lowRISC/opentitan/blob/8b675981e9a9f2a7ebadc2f7964e4b2437bacce6/hw/dv/tools/xcelium/common_cov_excl.tcl#L9
I somehow was not able to open the coverage html/gui in my workstation..

Reply:
sure let me see if I can get that done later today